src/main/java/com/lightstep/tracer/shared/AbstractTracer.java

```diff
@@ -521,13 +523,15 @@ protected void addTracerTag(String key, Object value) {
     /**
      * Internal logging.
      */
+    @SuppressWarnings("WeakerAccess")
     protected void debug(String s) {
         debug(s, null);
     }
 
     /**
      * Internal logging.
      */
+    @SuppressWarnings({"WeakerAccess", "SameParameterValue"})
```

Comment:
From IntelliJ's perspective there is only one caller of this method, AbstractTracer.debug(String), and it always passes null. I kind of wish adding the "WeakerAccess" suppression would transitively remove the "SameParameterValue" warning since by added WeakerAccess we are saying there are other callers, outside of this project, but it doesn't work that way.